components/Base/ButtonItem.tsx

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
-import { GestureResponderEvent, StyleSheet, TouchableOpacity, View, ViewStyle, TextStyle } from 'react-native';
+import { GestureResponderEvent, StyleSheet, TouchableOpacity, View, ViewStyle, TextStyle, StatusBar } from 'react-native';
 import useTheme from "../../hooks/useTheme";
 import { AppTheme } from "../../config/DefaultConfig";
 import ThemedText from '../UI/ThemedText';
 
 interface Props {
-    userName: string;
-    onChatPress?: (event: GestureResponderEvent) => void
+    label: string;
+    onButtonPress?: (event: GestureResponderEvent) => void
 };
 
-const ChatItem: React.FunctionComponent<Props> = ({
-    onChatPress,
-    userName,
+const ButtonItem: React.FunctionComponent<Props> = ({
+    onButtonPress,
+    label,
 }: Props) => {
     const theme: AppTheme = useTheme();
 
     return (
-        <TouchableOpacity onPress={onChatPress}>
+        <TouchableOpacity onPress={onButtonPress}>
+            <View>
+                <StatusBar
+                    translucent
+                    barStyle={theme ? "dark-content" : "light-content"}
```

Comment:
I don't understand the logic here @yashkapoor8710

Reply:
I have remove this